data/pigui/mainmenu.lua

```diff
@@ -0,0 +1,213 @@
+-- Copyright © 2008-2018 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Engine = import('Engine')
+local Game = import('Game')
+local Ship = import("Ship")
+local ShipDef = import("ShipDef")
+local SystemPath = import("SystemPath")
+local Equipment = import("Equipment")
+local Format = import("Format")
+local ui = import('pigui/pigui.lua')
+local Vector = import('Vector')
+local Event = import('Event')
+local Lang = import("Lang")
+local lc = Lang.GetResource("core")
+local lui = Lang.GetResource("ui-core")
+local qlc = Lang.GetResource("quitconfirmation-core")
+local elc = Lang.GetResource("equipment-core")
+local clc = Lang.GetResource("commodity")
+
+local cargo = Equipment.cargo
+local misc = Equipment.misc
+local laser = Equipment.laser
+local hyperspace = Equipment.hyperspace
+
+local player = nil
+local colors = ui.theme.colors
+local pionillium = ui.fonts.pionillium
+local icons = ui.theme.icons
+
+local mainButtonSize = Vector(400,46) * (ui.screenHeight / 1200)
+local dialogButtonSize = Vector(150,46) * (ui.screenHeight / 1200)
+local mainButtonFontSize = 36 * (ui.screenHeight / 1200)
+
+local showQuitConfirm = false
+local quitConfirmMsg
+local max_flavours = 19
+
+local startLocations = {
+	{['name']=lui.START_AT_EARTH,
+	 ['desc']=lc.START_AT_EARTH_DESC,	 
+	 ['location']=SystemPath.New(0,0,0,0,6,48600),
+	 ['shipType']='sinonatrix',['money']=100,hyperdrive=true,
```

Comment:
just for consistency's sake, could you make this ['hyperdrive']?

Reply:
fixed.